src/pages/lib-v1/auth/auth-events/q/platform/[platform].mdx

```diff
@@ -3,36 +3,8 @@ export const meta = {
   description: `Listen to various auth events`
 };
 
-AWS Cognito Auth Plugin sends important events through Amplify Hub.
+AWS Cognito Auth Service sends important events through Amplify Hub.
 
-import ios_maintenance from '/src/fragments/lib-v1/ios-maintenance.mdx';
```

Comment:
It looks like at least Android is using this page on the `lib-v1` route. Is it safe to remove these fragments? Make sure to double check the directory in `directory.mjs`: https://docs.amplify.aws/lib-v1/auth/auth-events/q/platform/android/